android/tangram/src/com/mapzen/tangram/HttpHandler.java

```diff
@@ -0,0 +1,58 @@
+package com.mapzen.tangram;
+
+import com.squareup.okhttp.Cache;
+import com.squareup.okhttp.Callback;
+import com.squareup.okhttp.OkHttpClient;
+import com.squareup.okhttp.Request;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public class HttpHandler {
+
+    private OkHttpClient okClient;
+    private Request.Builder okRequestBuilder;
+
+    public HttpHandler() {
+        okRequestBuilder = new Request.Builder();
+        okClient = new OkHttpClient();
+        okClient.setConnectTimeout(10, TimeUnit.SECONDS);
+        okClient.setReadTimeout(30, TimeUnit.SECONDS);
```

Comment:
Should we provide setters for these timeouts also, like we do for cache?
Nothing urgent though. (will merge it like this for now).